depends/packages/twisted.mk

```diff
@@ -0,0 +1,10 @@
+package=twisted
+$(package)_version=15.0.0
+$(package)_download_path=https://pypi.python.org/packages/source/T/Twisted
+$(package)_file_name=Twisted-$($(package)_version).tar.bz2
+$(package)_sha256_hash=ac609262253057cf2aeb9dc049ba7877d646f31b4caef06a50189a023df46b51
+$(package)_dependencies=native_python2 native_zope.interface
+
+define $(package)_stage_cmds
+  $($(package)_prefix)/native/bin/python setup.py install --prefix=$($(package)_staging_prefix_dir)
```

Comment:
But doesn't it not matter for twisted as it is pure Python? For something like depends/packages/pyqt4.mk it would make a difference, though. So would I do something like:

```
$($(package)_prefix)/native/bin/python setup.py install --prefix=$($(package)_staging_prefix_dir) CC=$($(package)_cc)
```

and so on with all the other variables, like the way $(1)_autoconf is setup in funcs.mk? Or is there some better way to be doing it?